lpcompiler.pas

```diff
@@ -2740,6 +2787,44 @@ function TLapeCompiler.ParseExpression(ReturnOn: EParserTokenSet = []; FirstNext
             Dec(InExpr);
           end;
 
+        tk_sym_BracketOpen:
+          if (_LastNode = _Var) then
+          begin
+            PopOpStack(op_Invoke);
```

Comment:
`op_Invoke` doesn't necessarily have to stay an alias for `op_Index` in the future. Also, at this point, we are not invoking it; only when it's a property, there will some invocation in the background.

However, as this is used for operator precedence, it would make sense to keep this consistent between properties and array indices.